Sources/TSCUtility/FSWatch.swift

```diff
@@ -93,17 +95,37 @@ private protocol _FileWatcher {
     func stop()
 }
 
-#if canImport(Glibc)
+#if os(OpenBSD)
+extension FSWatch._WatcherDelegate: NoOpDelegate {}
+extension NoOp: _FileWatcher{}
+#elseif canImport(Glibc)
 extension FSWatch._WatcherDelegate: InotifyDelegate {}
 extension Inotify: _FileWatcher{}
 #elseif os(macOS)
 extension FSWatch._WatcherDelegate: FSEventStreamDelegate {}
 extension FSEventStream: _FileWatcher{}
+#else
+#error("Implementation required")
 #endif
 
 // MARK:- inotify
 
-#if canImport(Glibc)
+#if os(OpenBSD)
+
+public protocol NoOpDelegate {
+    func pathsDidReceiveEvent(_ paths: [AbsolutePath])
+}
+
+public final class NoOp {
```

Comment:
should this have `FSWatch` or similar in the name given how generic the term is?

Reply:
Renamed to NoOpWatcher and NoOpWatcherDelegate.